site/source/docs/introducing_emscripten/about_emscripten.rst

```diff
@@ -4,66 +4,68 @@
+Emscripten is a complete :ref:`Open Source <emscripten-license>` compiler
+toolchain to WebAssembly. Using Emscripten you can:
+
+- Compile C and C++ code, or any other language that uses LLVM, into WebAssembly,
+  and run it on the Web, Node.js, or wasm runtimes.
+- Compile the C/C++ **runtimes** of other languages into WebAssembly, and then
+  run code in those other languages in an *indirect* way (for example, this has
+  been done for
+  `Python <https://github.com/iodide-project/pyodide>`_ and
+  `Lua <https://daurnimator.github.io/lua.vm.js/lua.vm.js.html>`_).
+
+Practically any **portable** C or C++ codebase can be compiled into WebAssembly
+using Emscripten, ranging from high performance games that need to render
+graphics, play sounds, and load and process files, through to application
+frameworks like Qt. Emscripten has already been used to convert a very long list
+of real-world codebases to WebAssembly, including commercial codebases like the
+`Unreal Engine 4 <https://blog.mozilla.org/blog/2014/03/12/mozilla-and-epic-preview-unreal-engine-4-running-in-firefox/>`_
+and the `Unity <https://blogs.unity3d.com/2018/08/15/webassembly-is-here/>`_ engine.
+For examples and demos, see the
+`community-maintained list on the wiki <https://github.com/emscripten-core/emscripten/wiki/Porting-Examples-and-Demos>`_.
+
+Emscripten generates small and fast code! Its default output format is
+`WebAssembly <http://webassembly.org/>`_ , a highly optimizable executable
+format, that runs almost as fast as native code, while being portable and safe.
+Emscripten does a lot of optimization work for you automatically, by careful
+integration with LLVM,
+`Binaryen <https://github.com/WebAssembly/binaryen>`_,
+`Closure Compiler <https://developers.google.com/closure/compiler>`_, and other
+tools.
 
 .. _about-emscripten-toolchain:
 
 Emscripten Toolchain
 ====================
 
-A high level view of the Emscripten toolchain is given below. The main tool is the :ref:`emccdoc`. This is a drop-in replacement for a standard compiler like *gcc*.
-
-.. image:: EmscriptenToolchain.png
-
-*Emcc* uses :term:`Clang` and LLVM to compile to wasm or asm.js. Emscripten emits JavaScript that can run that compiled code and provide the necessary runtime support. That JavaScript can be executed by :term:`node.js`, or from within HTML in a browser.
-
-The :ref:`emsdk` is used to manage multiple SDKs and tools, and to specify the particular SDK/set of tools currently being used to compile code (the :term:`Active Tool/SDK`). It can even "install" (download and build) the latest toolchain from GitHub!
-
-*Emsdk* writes the "active" configuration to the :ref:`compiler-configuration-file`. This file is used by *emcc* to get the correct current toolchain for building.
+A high level view of the Emscripten toolchain is given below. The main tool is
+the :ref:`emccdoc`. This is a drop-in replacement for a standard compiler like *gcc* or *clang*.
```

Comment:
should this say "emcc" instead of "the emcc"? actually I guess I don't know what word will actually be rendered here, and the GH preview doesn't really show it.

Reply:
That turns into `The main tool is the Emscripten Compiler Frontend (emcc).`